modules/eks/actions-runner-controller/variables.tf

```diff
@@ -146,29 +146,7 @@ variable "runners" {
   ```
   EOT
 
-  type = map(object({
```

Comment:
Why did you changed structured type to `any`

Reply:
because the runners.resources.limits value will not match between the runners if we want to enable ephemeral-storage limits for the dind runner

Reply:
added optional feature, added by Terraform version 1.3.x. Allows us to keep the map object